### PR TITLE
debug mode for seeing individual geometries of the leaves.

### PR DESCRIPTION
**Pull request by saraybakker1**

### CODE REVIEW

File: examples/panda_debug.py [hunk 1]
```diff
@@ -0,0 +1,236 @@
...
+    x_goal_1 = [0.1, 0.0, 0.0]
+
+    (env, goal) = initalize_environment(render)
+    planner = set_planner(goal)
+    action = np.zeros(7)
+    ob, *_ = env.step(action)
+
+    # specify the list of geometry leaf names that you would like to observe:
+    leaf_names = ["obst_0_panda_link9_leaf"]  #specify just one leaf name
+    all_leaf_names = list(planner.leaves.keys())  #list of all possible leave names
```
[the remark this answers is not in the document]
Comment (saraybakker1) [reply examples/panda_debug.py]: fixed

File: examples/panda_debug.py [hunk 2]
```diff
@@ -0,0 +1,236 @@
...
+        # mapping.concretize()
+        # unpulled_geometry=leave._geo  #weight_goal_0=goal.sub_goals()[0].weight() must be inputted somewhere.
+        # unpulled_geometry.concretize()
+
+    # list of possible input keys: To Do for Saray
+    # params = list(pulled_geometry._vars.parameters().keys())
+    # params_list = ["x_obst_"+str(i) for i in range(nr_obst)]+["radius_obst_"+str(i) for i in range(nr_obst)]+\
+    #               ["radius_body_panda_link"+str(i) for i in collision_links]+\
+    #               ["x_goal_"+str(i) for i in range(nr_goal)]+["weight_goal_"+str(i) for i in range(nr_goal)]
+
```
[the remark this answers is not in the document]
Comment (saraybakker1) [reply examples/panda_debug.py]: is it clear? :)

[the remark this answers is not in the document]
Comment (maxspahn) [reply examples/panda_debug.py]: yeah, looks good.

**Review by maxspahn — CHANGES_REQUESTED** [review]
It looks great, @saraybakker1 

However, this PR adds two relatively big example cases. I would prefer to have a smaller case to showcase how debugging works. I will add one for a point robot. In your specific use-case, you can then still use the debugging for the panda, but there are already so many examples in this folder.

I will do the changes now.

**Comment by saraybakker1** [comment]
Hi @maxspahn, that is also fine. If you would like, please add all examples (including the more complex ones) in the MultiFabrics git, then we can still use them

**Review by maxspahn — APPROVED** [review]

**Pull request merged**